package.json

```diff
@@ -17,8 +17,21 @@
     ],
     "scripts": {
         "build": "tsc --project tsconfig.build.json",
-        "test": "jest"
+        "format": "prettier --write \"{src/**/*.ts,jest.config.js}\"",
+        "prepare": "simple-git-hooks",
+        "pre:commit": "lint-staged",
+        "test": "npm run test:specs && npm run test:format",
```

Comment:
Let's run the prettier first so that if we have styling issues we don't fire any tests at all
```
npm run test:format && npm run test:specs
```